src/main/java/me/xmrvizzy/skyblocker/skyblock/dwarven/DwarvenHud.java

```diff
@@ -26,7 +26,8 @@ public class DwarvenHud {
 
     public static final List<Pattern> COMMISSIONS = Stream.of(
                     "(?:Titanium|Mithril|Hard Stone) Miner",
-                    "(?:Ice Walker|Goblin|Goblin Raid|Automaton|Sludge|Team Treasurite Member|Yog|Boss Corleone|Thyst) Slayer",
+                    "Golden Goblin Slayer",
+                    "(?:Ice Walker|(?<!Golden )Goblin|Goblin Raid|Automaton|Sludge|Team Treasurite Member|Yog|Boss Corleone|Thyst) Slayer",
```

Comment:
```suggestion
                    "(?:Ice Walker|Golden Goblin|(?<!Golden )Goblin|Goblin Raid|Automaton|Sludge|Team Treasurite Member|Yog|Boss Corleone|Thyst) Slayer",
```
We don't need to add a separate line